src/main/scala/com/fulcrumgenomics/vcf/filtration/ReadEndSomaticVariantFilter.scala

```diff
@@ -27,46 +27,49 @@ package com.fulcrumgenomics.vcf.filtration
 import java.lang.Math.{min, pow}
 
 import com.fulcrumgenomics.bam.{Bams, BaseEntry, Pileup, PileupEntry}
-import com.fulcrumgenomics.commons.util.LazyLogging
+import com.fulcrumgenomics.commons.util.{LazyLogging, Logger}
 import com.fulcrumgenomics.util.NumericTypes.{LogProbability => LnProb}
 import com.fulcrumgenomics.util.Sequences
+import com.fulcrumgenomics.vcf.api.Allele.SimpleAllele
 import com.fulcrumgenomics.vcf.api.{VcfCount, VcfFieldType, VcfInfoHeader, _}
+import com.fulcrumgenomics.vcf.filtration.ReadEndSomaticVariantFilter.{isPointMismatch, priors}
```

Comment:
Would prefer not to import these like this.  If called from within something implementing that trait you don't need this.  And if from outside the trait, better to call with `ReadEndSomaticVariantFilter.priors(...)` to make it clearer.